operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -39,25 +39,32 @@ kiali_defaults:
 
   external_services:
     grafana:
-      api_key: ""
+      auth:
+        ca_file: ""
+        insecure_skip_verify: false
+        password: ""
+        token: ""
+        type: "none"
+        use_kiali_token: false
+        username: ""
       display_link: true
-      in_cluster: true
-      insecure_skip_verify: false
-      namespace: ""
-      password: ""
-      service: "grafana"
+      in_cluster_url: "http://grafana.istio-system:3000"
```

Comment:
This is not good - it assumes grafana is in istio-system which won't be the case in multitenancy. 

We need to set this default to `""` empty string and define this in roles/kiali-deploy/tasks/main.yml - it should set this to `http://grafana.{{ kiali_vars.istio-namespace }}:3000`

Reply:
@jmazzitelli I prefer to have a default that works in the most common cases, and having overrides performed in less common cases. For the multi-tenant / Maistra scenario, I assume should be able to override any config ; is that not the case?

Reply:
BTW what you're suggesting is more or less what I suggesting above (2 days ago) : https://github.com/kiali/kiali/pull/1120#discussion_r292786599
So if that works, sure we can do that instead